libs/blocks/src/lib/stat/numbers-with-title/index.tsx

```diff
@@ -16,11 +16,9 @@ export const NumbersWithTitle: React.FC<NumbersWithTitleProps> = ({
     <Section className="bg-solid-slate-75 py-general-4xl">
       <FluidContainer className="flex flex-col gap-gap-3xl bg-solid-slate-75">
         <div className="flex flex-col items-center gap-gap-lg text-center lg:gap-gap-xl">
-          {title && (
-            <Heading.H2 className="text-solid-slate-1400">{title}</Heading.H2>
```

Comment:
I'm not sure if we need this here. I think we don't need this class as the default Heading should be black color. Please correct me if I am wrong.

Reply:
if the color is thesame with the default one, please go and remove it @jia-deriv